pootle/apps/pootle_terminology/receivers.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from pootle.core.delegate import terminology
+from pootle_store.constants import TRANSLATED
+from pootle_store.models import Unit
+
+
+@receiver(post_save, sender=Unit)
+def handle_unit_save(**kwargs):
+    unit = kwargs["instance"]
+    if unit.state != TRANSLATED:
+        return
+    is_terminology = (
```

Comment:
I would rename this to `is_in_terminology_tp`. Otherwise can be misleading since each regular project can have a `pootle-terminology` store which holds project-specific terminology, and we are leaving out that other terminology.